### PR TITLE
Move LICENCE info into dedicated file

### PR DESCRIPTION
**Pull request by katstevens**
Adds the MIT Licence as per our other libraries (see dmutils example: https://github.com/alphagov/digitalmarketplace-utils/blob/master/LICENCE).

### CODE REVIEW

**Comment by katstevens** [comment]
Docs are covered? `...this software and associated documentation files...`

**Comment by lfdebrux** [comment]
The GDS Way recommends docs are covered under a seperate licence https://gds-way.cloudapps.digital/standards/publish-opensource-code.html#how-to-publish-open-source-code

**Comment by katstevens** [comment]
Sure. In that case I'll link it up properly as per the GDS way example. 

```
## Licence

Unless stated otherwise, the codebase is released under [the MIT License][mit].
This covers both the codebase and any sample code in the documentation.

The documentation is [© Crown copyright][copyright] and available under the terms
of the [Open Government 3.0][ogl] licence.

[mit]: LICENCE
[copyright]: http://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/
[ogl]: http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/
```

File: README.md [hunk 1]
```diff
@@ -53,7 +53,14 @@ Find out how to [update with npm](docs/installation/updating-with-npm.md).
...
-documentation is &copy; Crown copyright and available under the terms of the
-Open Government 3.0 licence.
+Unless stated otherwise, the codebase is released under [the MIT License][mit].
+This covers both the codebase and any sample code in the documentation.
+
+The documentation is [&copy; Crown copyright][copyright] and available under the terms
+of the [Open Government 3.0][ogl] licence.
+
+[rvm]: https://www.ruby-lang.org/en/documentation/installation/#managers
+[bundler]: http://bundler.io/
```
Comment (lfdebrux) [inline README.md]: Don't think we're using ruby (yet) 😆

Comment (katstevens) [reply README.md]: ARGH I definitely removed that before committing, I'm sure I did 😹

**Pull request merged**